File: Homework Assignments/String Manipulation/str_1.py
# Find if the input string has all the alphabets; capital or small alike.

strOrig = ""
strNew = ""

charAsc = ''

total = 0
count = 0

strOrig = input("Enter a string: ")
strOrig = strOrig.lower()

for count in range(0, len(strOrig)):
    
    charAsc = strOrig[count]
    
    if ord(charAsc) >= 97 and ord(charAsc) <= 122:
        if charAsc not in strNew:
            strNew = strNew + charAsc
            total = total + ord(charAsc)

if total == 2847:
    print("The string has all the alphabets.")
else:
    print("The string does not have all the alphabets.")

File: Homework Assignments/String Manipulation/str_1.vb
' Find if the input string has all the alphabets; capital or small alike.

Module Module1

    Sub Main()
        Dim strOrig, strNew As String
        Dim charAsc As Char
        Dim total, count As Integer

        strOrig = ""
        strNew = ""

        charAsc = ""

        total = 0
        count = 0

        Console.Write("Enter a string: ")
        strOrig = Console.ReadLine()
        strOrig = LCase(strOrig)

        For count = 1 To Len(strOrig)

            charAsc = Mid(strOrig, count, 1)

            If charAsc >= "a" And charAsc <= "z" Then
                If InStr(strNew, charAsc) = 0 Then
                    strNew = strNew & charAsc
                    total = total + Asc(charAsc)
                End If
            End If
        Next

        If total = 2847 Then
            Console.WriteLine("The string has all the alphabets.")
        Else
            Console.WriteLine("The string does not have all the alphabets.")
        End If

        Console.ReadKey()
    End Sub

End Module
